app/Console/Commands/ModdingRankCommand.php

```diff
@@ -69,7 +69,7 @@ private function rankAll($modeInt)
         $toBeRankedQuery = Beatmapset::qualified()
             ->withoutTrashed()
             ->withModesForRanking($modeInt)
-            ->where('queued_at', '<', now()->subDays(7));
+            ->where('queued_at', '<', now()->subDays(Beatmapset::MINIMUM_DAYS_FOR_RANKING));
```

Comment:
May as well also move `Beatmapset::MINIMUM_DAYS_FOR_RANKING` to config while we're at it?